runtime-modules/membership/src/lib.rs

```diff
@@ -182,6 +186,35 @@ pub struct BuyMembershipParameters<AccountId, MemberId> {
     pub referrer_id: Option<MemberId>,
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, TypeInfo)]
+pub struct GiftMembershipParameters<AccountId, Balance> {
+    /// New member root account.
+    pub root_account: AccountId,
+
+    /// New member controller account.
+    pub controller_account: AccountId,
+
+    /// New member handle.
+    pub handle: Option<Vec<u8>>,
+
+    /// Metadata concerning new member.
+    pub metadata: Vec<u8>,
```

Comment:
This could potentially open up doors for unbounded input type attacks, shouldn't we use [BoundedVec](https://paritytech.github.io/substrate/master/frame_support/struct.BoundedVec.html)